_posts/en/newsletters/2023-12-06-newsletter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Bitcoin Optech Newsletter #280'
+permalink: /en/newsletters/2023/12/06/
+name: 2023-12-06-newsletter
+slug: 2023-12-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Cluster mempool discussion:** Bitcoin Core developers working on
+  [cluster mempool][topic cluster mempool] started a [working
+  group][wg-cluster-mempool] (WG) on the Delving Bitcoin forum. Cluster
+  mempool is a proposal to make it much easier to compare the
+  profitability of different valid orderings of transactions.  A valid
+  ordering of Bitcoin transactions requires parent transactions get
+  confirmed before their children, either by putting the parent in an
+  earlier block than any of its children, or earlier in the same block.
+  _Clusters_ of one or more related transactions are designed to be easy
+  to include into feerate-sorted _chunks_ that are each independent of
+  the other chunks. For example, we might split a mempool into 100
+  chunks each about 4 million weight in size, each of which could be
+  mined successively into a block.
+
+    By comparing different lists of chunks, we can determine which would
```

Comment:
"Chunkings across multiple clusters" would only occur when we are trying to determine whether a replacement is strictly better than our current mempool content. E.g. a replacement transaction could combine two previously unconnected clusters, and then we’d have to create combined feerate diagram across both these clusters to compare the state with the two separate clusters to the state after the replacement.